CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v0.2.0
```

Comment:
we typically save changes to CHANGELOG for releases.  please revert this.

Reply:
Reverted: https://github.com/deislabs/akri/pull/206/commits/365dd096f6bfcc7b2372962388834a9dddf030b4